Specify SiteAccess for translations data migrations: --siteaccess section (#2248)
ibexa/documentation-developer: docs/content_management/data_migration/exporting_data.md

!!! caution

>     [`--siteaccess` option](#siteaccess) usage can be relevant when multiple languages or multiple repositories are used.

Reviewer: ```suggestion
    To prevent data loss, when there are multiple languages or multiple repositories to migrate, make sure to use [`--siteaccess` option](#siteaccess)  to migrate them individually.
```

>     [`--siteaccess` option](#siteaccess) usage can be relevant when multiple languages or multiple repositories are used.

Author: ```suggestion
    [`--siteaccess` option](#siteaccess) usage can be relevant when multiple languages or multiple repositories are used.
    Translation could be lost if a SiteAccess not supporting all your languages is used.
```

!!! caution

    [`--siteaccess` option](#siteaccess) usage can be relevant when multiple languages or multiple repositories are used.

>     Translations could be lost if a SiteAccess not supporting all your languages is used.

Author: ```suggestion
    To prevent translation loss, a SiteAccess having all the languages must be use (often a Back Office one).
```

Author: ```suggestion
    To prevent translation loss, it is recommended that you use the SiteAccess that has all the languages used in your implementation, most likely the Back Office one.
```

|`segment_group`|&#10004;|&#10004;|&#10004;|
|`company`|&#10004;|||

## siteaccess

> The optional `--siteaccess` option enables to export (or import) in a SiteAccess configuration's context.

Author: ```suggestion
The optional `--siteaccess` option enables to export (or import) in a SiteAccess configuration's context.
If not provided, the [default SiteAccess](multisite_configuration.md#default-siteaccess) is used.
```

It is recommended to use the SiteAccess of the Back Office of the targeted repository.

To precise the SiteAccess can be mandatory when using several languages.

> You must export with the SiteAccess supporting all the languages, or migration skips non-supported languages' translations.

Author: ```suggestion
Export and import command only works with languages supported by the SiteAccess.
You must export and import with the SiteAccess supporting all used languages to not loose translations.
```

|`segment_group`|&#10004;|&#10004;|&#10004;|
|`company`|&#10004;|||

## siteaccess

> The optional `--siteaccess` option enables to export (or import) in a SiteAccess configuration's context.

Reviewer: ```suggestion
The optional `--siteaccess` option enables you to export (or import) data in a SiteAccess configuration's context.
```

If not provided, the [default SiteAccess](multisite_configuration.md#default-siteaccess) is used.

> It is recommended to use the SiteAccess of the Back Office of the targeted repository.

Reviewer: ```suggestion
It is recommended that you use the SiteAccess of the  target repository's Back Office.
```

(To get rid of the "of the" tepetition)

> To precise the SiteAccess can be mandatory when using several languages.

Author: ```suggestion
To precise the SiteAccess can be mandatory when using [several languages](languages.md#using-siteaccesses-for-handling-translations).
```

> To precise the SiteAccess can be mandatory when using [several languages](languages.md#using-siteaccesses-for-handling-translations).

Reviewer: ```suggestion
Specifying the SiteAccess can be mandatory, for example, when you use several SiteAccesses to handle [several languages](languages.md#using-siteaccesses-for-handling-translations).
```

Export and import commands only work with languages supported by the context SiteAccess.

> You must export and import with the SiteAccess supporting all used languages to not loose translations.

Reviewer: ```suggestion
You must export and import with the SiteAccess supporting all used languages to preserve translations.
```
